features/step_definitions/asstertion_stpes.rb

```diff
@@ -1,4 +1,9 @@
 Then("I should see {string}") do |content|
```

Comment:
Two typos in the file name

Reply:
good call!!! we were just making sure you did a really good review ;)